### PR TITLE
Remove duplicated comments

### PR DESCRIPTION
**Pull request by san650**

### CODE REVIEW

File: packages/ember-glimmer/lib/component.js [hunk 1]
```diff
@@ -308,24 +308,6 @@ const Component = CoreView.extend(
      */
 
     /**
-     Called when the attributes passed into the component have been updated.
-     Called both during the initial render of a container and during a rerender.
-     Can be used in place of an observer; code placed here will be executed
-     every time any attribute updates.
-     @event didReceiveAttrs
```
Comment (locks) [inline packages/ember-glimmer/lib/component.js]: where did `didReceiveAttrs` go?

**Review by locks — CHANGES_REQUESTED** [review]

File: packages/ember-glimmer/lib/component.js (hunk 1 above)
Comment (locks) [inline packages/ember-glimmer/lib/component.js]: nevermind, hidden in the fold above.

**Review by locks — APPROVED** [review]

**Comment by locks** [comment]
Thanks for your contribution!

@homu r+

**Comment by rwjblue** [comment]
These were not duplicated (as far as I can tell), the documentation is for them as both an event and a method.

**Comment by locks** [comment]
Ah, damnit :( I triple checked and missed it. Reverting it, apologies @san650.

**Comment by san650** [comment]
@rwjblue @locks You're right, I didn't notice they were different.

Sorry for all the trouble!

**Pull request merged**